docker/keycloak/README.md

```diff
@@ -0,0 +1,76 @@
+# Keycloak infrastructure
+
+WOO Publications supports OpenID Connect as an authentication protocol. Keycloak is an example of an Identity Provider that supports OIDC.
+
+We include a compose stack for development and testing/CI purposes. This is **NOT** suitable for
+production usage.
+
+## docker compose
+
+Start a Keycloak instance in your local environment from the parent directory:
+
+```bash
+docker compose -f docker/docker-compose.keycloak.yml up -d
+```
+
+This brings up Keycloak, the admin interface is accessible at http://localhost:8080/. You can now
+log in with the `admin`/`admin` credentials.
+
+In order to allow access to Keycloak via the same hostname via the WOO Publications backend container and the browser, add the following entry to your `/etc/hosts` file:
+
+```
+127.0.0.1 keycloak.woo-publications.local
+```
+
+## Testing
+
+This realm is used in the integration tests. We re-record the network traffic periodically (using
+VCR.py). The primary reason this setup exists, is for automated testing reasons.
+
+### Test data
+
+**Clients**
+
+- Client ID: `test-userinfo-jwt`, secret `ktGlGUELd1FR7dTXc84L7dJzUTjCtw9S`
+
+  Configured to return the user info as a JWT rather than JSON response.
+
+- Client ID: `testid`, secret: `7DB3KUAAizYCcmZufpHRVOcD0TOkNO3I`
+
+**Users**
+
+- `testuser` / `testuser`, has the `bsn`, `kvk`, `name_qualifier`, `legalSubjectID` and
```

Comment:
mental note to clean this up from the fixture, as these are not relevant for WOO